pycon/settings.py

```diff
@@ -8,18 +8,15 @@
 TEMPLATE_DEBUG = DEBUG
 
 ADMINS = (
-    ('dvd', 'dvd@gnx.it'),
-    ('c8e', 'carlo.miron@gmail.com'),
-    ('yakky', 'github@spalletti.it'),
+    ('alexsavio', 'alexsavio@gmail.com'),
```

Comment:
Ok.

Alexandre Manhães Savio  
PhD, Medical Imaging, Machine Learning
Grupo de Inteligencia Computacional UPV/EHU
home | email: alexsavio@gmail.com

From: fpliger <notifications@github.com>
Reply: EuroPython/epcon <reply@reply.github.com>>
Date: 14 Feb 2015 at 09:52:03
To: EuroPython/epcon <epcon@noreply.github.com>>
Cc: Alexandre M. S. <alexsavio@gmail.com>>
Subject:  Re: [epcon] Fix p3/templates not loading. (#8)  

In pycon/settings.py:

> @@ -8,18 +8,15 @@
>  TEMPLATE_DEBUG = DEBUG
>   
>  ADMINS = (
> -    ('dvd', 'dvd@gnx.it'),
> -    ('c8e', 'carlo.miron@gmail.com'),
> -    ('yakky', 'github@spalletti.it'),
> +    ('alexsavio', 'alexsavio@gmail.com'),
could you add also other web WG devs? [at least the active ones]  

—
Reply to this email directly or view it on GitHub.